src/Datadog.Trace.ClrProfiler.Native/integration_loader.cpp

```diff
@@ -12,33 +12,30 @@ namespace trace
 
 using json = nlohmann::json;
 
-std::vector<Integration> LoadIntegrationsFromEnvironment()
+void LoadIntegrationsFromEnvironment(std::vector<IntegrationMethod>& integrationMethods, const bool isCallTargetEnabled,
+                                     const bool isNetstandardEnabled,
+                                     const std::vector<WSTRING> disabledIntegrationNames)
```

Comment:
pass `disabledIntegrationNames` by `const &` no?

Reply:
done in 1aa6b9d